packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -13,7 +13,9 @@
 #import <EXAV/EXAV.h>
 #import <EXAV/EXAVPlayerData.h>
 #import <EXAV/EXVideoView.h>
+#if !TARGET_OS_TV
```

Comment:
in expo-av, do you have any better solutions than these if checks?
it would be hard to maintain those checks from time to time, especially we don't have a ci to build tv variants. we will not know if we break anything on tv during new feature development or refactoring.

some idea includes adding an AudioRecordingDelegate to put permission requests, audio recordings, and everything related inside the abstract class. ios and tv could have their own implementations.

Reply:
I'll have a look at this, to see if there is a more maintainable way to support this.

We will have CI for TV once I'm done with the TV changes.

For now, I'll remove expo-av from the list of supported modules, and add that in a separate PR.